README.md

```diff
@@ -422,13 +426,17 @@ attr <device> rhasspyName bulb,leiling light,chandelier
 It's also possible to have the same name for different FHEM-devices. Just make sure they are located in different _rooms_ (e.g. by setting the *rhasspyRoom* attribute).
 
 ### Attribute *rhasspyRoom*
-You can add an attribute `rhasspyRoom` to the device to tell Rhasspy in which physical (or logical) room the device is. If omitted, (alexaRoom or) standard FHEM _room_ attribute is used. If this is also not provided, it belongs to the "default room" as set in _define_\
-This is useful to speak commands without a room. If there is a device *bulb* and it's *rhasspyRoom*-attribute is equal to the siteId of your satellite, it's enough to say "Bulb on" and the bulb in the room the command is spoken will be turned on.
+You can add an attribute `rhasspyRoom` to the device to tell Rhasspy in which physical (or logical) room the device is. If omitted, (_alexaRoom_ or) standard FHEM _room_ attribute is used. If this is also not provided, it belongs to the "default room" as set in _define_\
+This is useful to speak commands without explicitely also naming a room. If there is a device *bulb* and it's *rhasspyRoom*-attribute is equal to the _siteId_ of your satellite, it's enough to say "Bulb on" and the bulb in the room the command is spoken will be turned on.
 `rhasspyRoom` also accepts a comma-separated list.
 
 Example:
 ```attr <device> rhasspyRoom livingroom```
 
+Additional remarks:
+* If siteId follows the grouping conventions in *Rhassy* ( *roomname.sub-satellite*), only *roomname* will be used as room info
+* You may use a special reading following the name convention 'siteId2room_*satellite*' to automatically redirect any siteId to the room indicated by *ReadingVal()* of this reading. See also myUtils example file (ins svn/contrib) for assinging suitable values by RHASSPY using a CustomIntent.
```

Comment:
room indicated → room, indicated 
of this reading → for this reading
assinging → assigning